CMakeLists.txt

```diff
@@ -38,13 +39,13 @@ if (POLICY CMP0074)
 endif()
 
 set(PROJECT_VERSION_MAJOR 3)
-set(PROJECT_VERSION_MINOR 4)
+set(PROJECT_VERSION_MINOR 5)
 set(PROJECT_VERSION_PATCH 0)
 set(PROJECT_VERSION
   "${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}")
 
 # Increment the PROJECT_SOVERSION every time you update VERSION_MINOR!
-SET(PROJECT_SOVERSION 6)
+SET(PROJECT_SOVERSION 7)
```

Comment:
@pford We use a custom additional `CASA_PROJECT_SOVERSION` for the imageanalysis library, which is based on the casa minor version rather than the casacore minor version. It needs to be bumped as well. (This was required for packaging.)

Reply:
I changed line 539 to "set( CASA_PROJECT_SOVERSION 6 )" for casa 6.6.